src/Common/tests/Scripts/Tools/net_providers.txt

```diff
@@ -0,0 +1,12 @@
+"{bdd9a83e-1929-5482-0d73-2fe5e1c0e16d}"
```

Comment:
I don't know how to add comments.
Using wildcard text names as in PerfView won't work with logman.